### ISSUE TITLE
Parsing issue with new style syntax on sterotype class

### ISSUE THREAD

**Issue by boerngen-schmidt** [plantuml/plantuml#1217]
I found a glitch in the parsing when using the new style syntax

[Not Working](https://www.plantuml.com/plantuml/uml/PO_12eCm38RlUOh0xXlkN4Jc6snFq6acmpQHj3qCyTrNXqYPkUN__tn9cX2jnkGTXF7DarNhiHC_2HD7d05pFUB74IfrqVBW21O04rQjXbtSeZVRZODAuhuL9uedqMol8__GGrAf_nddbmZEH__f4k6IIUQoBI-ORLKsLpGcK-holdkrYmrndp_-0W00)
```
component {
    BackgroundColor #red
    .foo { BackgroundColor #blue }
}
```
[Working](https://www.plantuml.com/plantuml/uml/TKyn3i8m3Dpz2alie397LKL_02y8hQbGKxjoam5L_Ji9L546y6JVdS_dPl5MVPWSBEEJPwjsmawcMPZOumeOw-PVZb2fyvO7Hx01c2LXDMHmbrviDmugWVjMd2WUH2ETLhBfFwLI_o6E3n5SCVK_TNSNwCjkqTQSzYJ5WROgXpCQ4othCkoroc13tCVttm00)
```
component {
    BackgroundColor #red
    .foo { 
       BackgroundColor #blue 
  }
}
```

Seems link linebreaks are nessesary, which they should not be.

**Comment by arnaudroques** [comment]
Thanks for the feedback, this is fixed in V1.2023.0

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed.
- Thanks @boerngen-schmidt for the report.
- Thanks PlantUML team for your work.
	
Regards.
